go/contacts/tasks.py

```diff
@@ -33,10 +33,13 @@ def delete_group(account_key, group_key):
     # We do this one at a time because we're already saving them one at a time
     # and the boilerplate for fetching batches without having them all sit in
     # memory is ugly.
-    for contact_key in group.backlinks.contacts():
-        contact = contact_store.get_contact_by_key(contact_key)
-        contact.groups.remove(group)
-        contact.save()
+    contacts_page = group.backlinks.contact_keys()
+    while contacts_page is not None:
+        for contact_key in contacts_page:
+            contact = contact_store.get_contact_by_key(contact_key)
+            contact.groups.remove(group)
+            contact.save()
+        contacts_page = contacts_page.next_page()
```

Comment:
We seem to do this all over the place

```python
while contacts_page is not None:
    for contact_key in contacts_page:
        # do stuff
     contacts_page = contacts_page.next_page()
```

Would it maybe simplify things if we had a way of iterating through the paginated contact keys as a normal iterable? For example, if the paginated result set had some `.all` method:

```python
for contact_key in group.backlinks.contact_keys().all():
    # do stuff here
```